tests/unit/disambiguation/test_receivers.py

```diff
@@ -82,8 +82,8 @@ def test_check_if_record_eligible_method():
         "full_name": "Grzegorz Jacenków"
     }]
 
-    assert _check_if_record_eligible(base_author, base_author) is not True
-    assert _check_if_record_eligible(base_author, author_new_affiliation) is \
+    assert _needs_beard_reprocessing(base_author, base_author) is not True
+    assert _needs_beard_reprocessing(base_author, author_new_affiliation) is \
```

Comment:
There's no need for `is True`: just `assert stuff_that_is_True`. Analogously, just `assert not stuff_that_is_False`.